user_manual/groupware/calendar.rst

```diff
@@ -128,29 +129,30 @@ cursor over the area when the event is taking place.
 
 .. only:: html
 
-  .. figure:: images/calendar_new-event_week.gif
+  .. figure:: images/calendar_new-event_week.png
 
-The month-view only requires a single click into the area of the specific day.
+The month-view only requires a single click into the area of the target day.
 
 .. only:: html
 
-  .. figure:: images/calendar_new-event_month.gif
+  .. figure:: images/calendar_new-event_month.png
 
 After that, you can type in the event's name (e.g. **Meeting with Lukas**), choose
 the calendar in which you want to choose the event (e.g. **Personal**, **Work**),
-check and concretize the time span or set the event as all-day event.
+check and concretize the time span or set the event as all-day event. Optionally
+you can specify a location and a description.
 
-If you want to edit advanced details such as the **Location**, a **Description**,
-**Attendees**, **Reminders** or to set the event as a repeating-event click on
-the ``More...``-button to open the advanced sidebar-editor.
+If you want to edit advanced details such as the **Attendees**, **Reminders** or 
```

Comment:
the **Attendees** or **Reminders**, or if you want to set the event as

Reply:
In my opinion, repeating "or" twice is not stylistically. I would leave this text sequence unchanged.